sdk/compute/arm-compute/src/computeManagementClient.ts

```diff
@@ -62,7 +63,6 @@ class ComputeManagementClient extends ComputeManagementClientContext {
     this.virtualMachineScaleSets = new operations.VirtualMachineScaleSets(this);
     this.virtualMachineScaleSetExtensions = new operations.VirtualMachineScaleSetExtensions(this);
     this.virtualMachineScaleSetRollingUpgrades = new operations.VirtualMachineScaleSetRollingUpgrades(this);
-    this.virtualMachineScaleSet = new operations.VirtualMachineScaleSetOperations(this);
```

Comment:
Is this operationgroup removed in the new api-version of the swagger spec? This would be a breaking change for customers.

Reply:
I have an explanation for this now.  Apparently a bug was introduced and fixed between these two commits on the same day:

- https://github.com/Azure/azure-rest-api-specs/commit/6a18ab3515151793abcad1266a53a94e5b812366
- https://github.com/Azure/azure-rest-api-specs/commit/d12a948c0480cba0df375dc030c9be5769415b00

_Somehow_ the package code was generated before the fix in the second commit, so an additional operation group was added to the code for the `convertToSinglePlacementGroup` operation by mistake.  When you generate using the spec in the second commit, the `convertToSinglePlacementGroup` operation gets added to the `operations.VirtualMachineScaleSets` class.

I'll regenerate with the bugged commit just so that the code is identical to what it was before.